vxquery-core/src/main/java/org/apache/vxquery/runtime/functions/jsonitem/ObjectConstructorScalarEvaluator.java

```diff
@@ -54,6 +52,8 @@ public ObjectConstructorScalarEvaluator(IHyracksTaskContext ctx, IScalarEvaluato
     @Override
     protected void evaluate(TaggedValuePointable[] args, IPointable result) throws SystemException {
         TaggedValuePointable key, value, qmc, tvp;
+        ArrayBackedValueStorage abvs = abvsPool.takeOne();
+        ArrayBackedValueStorage abvs1 = abvsPool.takeOne();
```

Comment:
Can we use more descriptive names for the abvs variables? How about abvsResult and abvsItem? Do those make sense?